### ISSUE TITLE
Pastastore sets log-level of Pastas to Error

### ISSUE THREAD

**Issue by rubencalje** [pastas/pastastore#116]
Not sure if this is intended behavior, but Paststore sets the log-level to Error, on importing pastastore (in yaml_interface.py).

Maybe it is better to remove these two logger-lines from `yaml_interface.py`:

```
ps.logger.setLevel("ERROR")

logging.basicConfig(level="INFO")
```

What do you think @dbrakenhoff ?

**Comment by dbrakenhoff** [comment]
This was probably somewhat intentional, because I wanted the YAML logging to be uninterrupted by other modules, but doing so by silencing pastas entirely when importing pastastore was probably not what I had in mind.

**Comment by dbrakenhoff** [comment]
fixed by 8f45fb9d66111eaf8112768ee22555b0ad3c2927

**Comment by rubencalje** [comment]
Thanks!
